git-cms-init

```diff
@@ -251,16 +251,13 @@ fi
 
 # check if a shared reference repository is available, otherwise set up a personal one
 if [ "$CMSSW_GIT_REFERENCE" = "" ]; then
-  case `hostname -f` in
-    *.cern.ch)
-      if [ -e /afs/cern.ch/cms/git-cmssw-mirror/cmssw.git ]; then
-        CMSSW_GIT_REFERENCE=/afs/cern.ch/cms/git-cmssw-mirror/cmssw.git
-      else
-        CMSSW_GIT_REFERENCE=~/.cmsgit-cache
-      fi
-    ;;
-    *) CMSSW_GIT_REFERENCE=~/.cmsgit-cache ;;
-  esac
+  if [ -e /cvmfs/cms-ib.cern.ch/git/cms-sw/cmssw.git ] ; then
```

Comment:
No, if /cvmfs is not available then it is not going to timeout. 
But if /cvmfs/cms-ib.cern.ch is not mounted then autofs is going to mount it